src/cljs_time/coerce.cljs

```diff
@@ -50,7 +50,7 @@
 (defn to-epoch
   "Convert `obj` to Unix epoch."
   [obj]
-  (some-> obj to-long (/ 1000)))
+  (some-> obj to-long (/ 1000) int))
```

Comment:
How about we just copy clj-time whole cloth? https://github.com/clj-time/clj-time/blob/master/src/clj_time/coerce.clj#L62-L66